pkg/subnet/local_test.go

```diff
@@ -226,9 +226,9 @@ func getTestClientFunc() (client.Client, error) {
 	c.On("Health", mock.Anything).Return(fakeHealthResponse, nil).Twice()
 	// Afterwards, change the VmId so that TestDeployToLocal has the correct ID to check
 	alteredFakeResponse := proto.Clone(fakeHealthResponse).(*rpcpb.HealthResponse) // new(rpcpb.HealthResponse)
-	alteredFakeResponse.ClusterInfo.CustomVms["bchain2"].VmId = testVMID
-	alteredFakeResponse.ClusterInfo.CustomVms["bchain2"].VmName = testVMName
-	alteredFakeResponse.ClusterInfo.CustomVms["bchain1"].VmName = "bchain1"
+	alteredFakeResponse.ClusterInfo.CustomChains["bchain2"].VmId = testVMID
+	alteredFakeResponse.ClusterInfo.CustomChains["bchain2"].ChainName = testVMName
```

Comment:
We might also want to rename `testVMName` to something like `testChainName`

Reply:
probably can be called testSubnetName, as it is being used as argument to DeployToLocalNetwork

Reply:
will change to testChainName